src/wallet/types.ts

```diff
@@ -124,8 +124,6 @@ export interface TxProposalCreateResponseData {
   success: boolean;
   txProposalId: string; // Id of the tx proposal
   inputs: TxProposalInputs[]; // Inputs data of the tx proposal
-  outputs: TxProposalOutputs[]; // Outputs data of the tx proposal
-  tokens: string[];
```

Comment:
These are not coming from the wallet-service:

https://github.com/HathorNetwork/hathor-wallet-service/blob/239b5a769b04a09714ca7fa673e99e064e91f471/packages/wallet-service/src/api/txProposalCreate.ts#L138-L145